nodejs/server.js

```diff
@@ -0,0 +1,254 @@
+const express = require('express');
+const cors = require('cors');
+const bodyParser = require('body-parser');
+const app = express();
+const crate = require('node-crate');
+const uniqid = require('uniqid');
+const sha1 = require('sha1');
+
+const request = require('request');
+
+app.use(cors());
+app.use(bodyParser.json({limit: '50mb'}));
+app.use(bodyParser.urlencoded({limit: '50mb', extended: true, parameterLimit: 1000000}));
+
+crate.connect('localhost', 4200);
+
+app.listen(8080, () => console.log('Example app listening on port 8080!'));
+
+app.get('/', (req, response) => response.json({
+    message: 'Welcome to the node.js example for crate db',
+    state: 'Server up and running'
+}));
+
+// get all posts
```

Comment:
This might be very very picky, but i think starting comments with capital letters is something that makes code look more serious!

```
// Get all posts
```